app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -26,26 +27,33 @@ import androidx.compose.ui.unit.dp
 import coil.compose.AsyncImage
 import com.github.se.cyrcle.R
 import com.github.se.cyrcle.model.parking.ParkingViewModel
+import com.github.se.cyrcle.model.review.ReviewViewModel
 import com.github.se.cyrcle.model.user.UserViewModel
 import com.github.se.cyrcle.ui.navigation.NavigationActions
 import com.github.se.cyrcle.ui.navigation.Screen
 import com.github.se.cyrcle.ui.theme.ColorLevel
 import com.github.se.cyrcle.ui.theme.atoms.Button
+import com.github.se.cyrcle.ui.theme.atoms.ScoreStars
 import com.github.se.cyrcle.ui.theme.atoms.Text
 import com.github.se.cyrcle.ui.theme.bold
 import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
 
+@SuppressLint("StateFlowValueCalledInComposition")
 @Composable
-fun CardScreen(
+fun ParkingDetailsScreen(
     navigationActions: NavigationActions,
     parkingViewModel: ParkingViewModel,
-    userViewModel: UserViewModel
+    userViewModel: UserViewModel,
+    reviewViewModel: ReviewViewModel
 ) {
   val selectedParking =
       parkingViewModel.selectedParking.collectAsState().value
           ?: return Text(stringResource(R.string.no_selected_parking_error))
-
   val userSignedIn = userViewModel.isSignedIn.collectAsState(false)
+  reviewViewModel.getReviewsByParking(selectedParking.uid)
+  selectedParking.nbReviews = reviewViewModel.parkingReviews.value.size
+  selectedParking.avgScore =
+      reviewViewModel.parkingReviews.value.sumOf { it.rating } / selectedParking.nbReviews
```

Comment:
Could it be a problem if selectedParking.nbReviews = 0 ?

Reply:
That shouldn't ever be possible, given the implementation, but putting a safe check could be a good idea. I'll add this